# 读取文件（观察数据集结构）

In [44]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
# 读取波士顿房价数据集
boston = pd.read_csv("boston.csv") 

# 查看数据集的前5行
print(boston.head())
print("******************************************************")#我是分割线
# 查看数据集的基本信息（可用于检测是否有空值/数值类型为多少）
print(boston.info())
print("******************************************************")#我也是分割线
# 查看数据集的统计描述
print(boston.describe())
print(xlabel)

      CRIM    ZN  INDUS  CHAS    NOX     RM   AGE     DIS  RAD  TAX  PIRATIO  \
0  0.00632  18.0   2.31     0  0.538  6.575  65.2  4.0900    1  296     15.3   
1  0.02731   0.0   7.07     0  0.469  6.421  78.9  4.9671    2  242     17.8   
2  0.02729   0.0   7.07     0  0.469  7.185  61.1  4.9671    2  242     17.8   
3  0.03237   0.0   2.18     0  0.458  6.998  45.8  6.0622    3  222     18.7   
4  0.06905   0.0   2.18     0  0.458  7.147  54.2  6.0622    3  222     18.7   

        B  LSTAT  MEDV  
0  396.90   4.98  24.0  
1  396.90   9.14  21.6  
2  392.83   4.03  34.7  
3  394.63   2.94  33.4  
4  396.90   5.33  36.2  
******************************************************
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 506 entries, 0 to 505
Data columns (total 14 columns):
 #   Column   Non-Null Count  Dtype  
---  ------   --------------  -----  
 0   CRIM     506 non-null    float64
 1   ZN       506 non-null    float64
 2   INDUS    506 non-null    float64
 3   CHAS     506 n

NameError: name 'xlabel' is not defined

## 数据集说明
波士顿房价数据集包含506个样本，每个样本有14个属性。这些属性包括：
- CRIM：城镇人均犯罪率
- ZN：占地面积超过25,000平方英尺的住宅用地比例
- INDUS：城镇非零售业务地区的比例
- CHAS：查尔斯河虚拟变量（如果临河为1，否则为0）
- NOX：一氧化氮浓度（每千万分之一）
- RM：住宅平均房间数
- AGE：1940年以前建成的自住单位比例
- DIS：到波士顿五个就业中心的加权距离
- RAD：径向公路的可达性指数
- TAX：每10,000美元的全额物业税率
- PTRATIO：城镇师生比例
- B：1000(Bk - 0.63)^2，其中Bk是城镇黑人比例
- LSTAT：人口中地位较低人群的百分比
- MEDV：自住房屋的中位数价值（单位：千美元）

###  绘制各特征与房价的散点图，观察它们之间的关系

### 加载数据并划分训练集/测试集